localization/english/event_localization/wc_horde_invasion_events_l_english.yml

```
@@ -0,0 +1,23 @@
+﻿l_english:
+ ### Northshire ###
+ wc_horde_invasion.0001.title:0 "The Fall of Northshire"
+ wc_horde_invasion.0001.desc:0 "As $c_northshire$ and its surrounding lands fall, our warriors descends into a frenzy of destruction. The siege's end has unleashed an orgy of violence; every building is pillaged, every human cut down in their tracks.\n\nThe final stand of the humans is at the abbey. As our warriors surround it, we tear an ancient tree from its roots in the gardens to use as a battering ram. Each thunderous crash against the abbey's doors mingles the roars of our war cries with the terrified screams of those inside. The timber groans, splinters flying, as the last barrier to our wrath buckles and bends.\n\nJust as the door begins to give way, poised to unleash our horde into their last haven, we prepare to extinguish the final flicker of resistance in $c_northshire$."
+ wc_horde_invasion.0001.a:0 "Lok'tar Ogar, for the Horde!"
+
+ wc_horde_invasion.0002.title:0 "The Last Stand of Northshire"
+ wc_horde_invasion.0002.desc:0 "The fall of $c_northshire$ is a nightmare come to life. Around us, the lands burn, and our people are mercilessly slaughtered. We, the last defenders, have retreated to the abbey, our final sanctuary.\n\nThrough the barricaded doors, we hear the orcs' fury grow louder. They have uprooted one of the sacred Holyoak trees from our gardens, now a tool of destruction against us. Each impact against the doors sends a wave of dread through our hearts. Our prayers for salvation are drowned out by the cacophony of battle - the orcs' battle cries, the splintering wood, the screams of the doomed.\n\nAs the doors begin to splinter and crack, the end nears. We brace ourselves, knowing that the flood of rage and steel is about to break through, and with it, the last light of $c_northshire$ will be extinguished."
```

```suggestion
 wc_horde_invasion.0002.desc:0 "The fall of $c_northshire$ is a nightmare come to life. The lands burn around us, and our people are mercilessly slaughtered. We last few defenders have retreated to the abbey, our final sanctuary.\n\nThrough the barricaded doors, we hear the orcs' fury grow louder. They have uprooted one of the sacred Holyoak trees from our gardens, transforming it into a tool of destruction. Each impact against the doors sends a wave of dread through our hearts. Our prayers for salvation are drowned out by the cacophony of battle - the [orcs'] battle cries, the splintering wood, the screams of the doomed.\n\nAs the doors begin to splinter and crack, the end nears. We brace ourselves, knowing that the flood of rage and steel is about to break through, and with it, the last light of $c_northshire$ will be extinguished."
```
please localize race